spec/install/gems/resolving_spec.rb

```diff
@@ -143,6 +143,31 @@
         expect(out).to_not include("rack-9001.0.0 requires ruby version > 9000")
         expect(the_bundle).to include_gems("rack 1.2")
       end
+
+      it "installs the older not platform specific version" do
+        build_repo4 do
+          build_gem "rack", "9001.0.0" do |s|
+            s.required_ruby_version = "> 9000"
+          end
+          build_gem "rack", "1.2" do |s|
+            s.platform = mingw
+            s.required_ruby_version = "> 9000"
+          end
+          build_gem "rack", "1.2"
+        end
+
+        simulate_platform mingw do
+          install_gemfile <<-G, :artifice => "compact_index_rate_limited", :env => { "BUNDLER_SPEC_GEM_REPO" => gem_repo4.to_s }
```

Comment:
The API mock here is very specific to test API fallback behavior when the modern API responds with 429 code (too many requests). I'd use the "compact_index" mock instead which is more general and I think more appropriate for this test.

```suggestion
          install_gemfile <<-G, :artifice => "compact_index", :env => { "BUNDLER_SPEC_GEM_REPO" => gem_repo4.to_s }
```

Reply:
You're right.
Sorry. I didn't care about it.